docs/stencil-docs/installing-stencil-cli/incompatibile-directives.mdx

```diff
@@ -0,0 +1,74 @@
+# Incompatible directives
+
+Since we have different versions of node sass running in stencil cli and production, there can be difference in the output of the css.
+This doc is to cover the differences and how to avoid them until the node sass fork is sunset.
+
+## Examples
+
+1. To avoid using / divisions in scss directives
+Since support of it was added later then node sass fork, it will break on production with error stating that this syntax is not supported.
```

Comment:
```suggestion
Since the support of SCSS directives came after the node-sass fork, the SCSS directives will break on production with an error stating that this syntax is not supported.
```